docs/source/appendix.rst

```diff
@@ -261,6 +261,24 @@ Asset Metadata
 .. autoclass:: zipline.assets.AssetConvertible
    :members:
 
+
+Trading Calendar API
+~~~~~~~~~~~~~~~~~~~~
```

Comment:
@jbredeche Needed to add TradingCalendar to the API reference, does this look sufficient to you?